picard/browser/browser.py

```diff
@@ -166,11 +174,17 @@ def do_GET(self):
             log.error('Browser integration failed handling request', exc_info=True)
             self._response(500, 'Unexpected request error')
 
+    def _fmt_message(self, format, args):
+        message = format % args
+        return "%s %s" % (self.address_string(), message.translate(self._control_char_table))
```

Comment:
Where is `_control_char_table` defined?

Reply:
In parent class, BaseHTTPRequestHandler,, perhaps we shouldn't rely on it, and use our own code. Not really sure it is needed, but it looks safer.

Reply:
It's internal, so there is a small risk it will break between Python versions. But I see how the parent class uses it, so when overriding the log methods it makes sense to apply the same handling.

Reply:
This line is a bit long, and there is a lot going on for each parameter. I'd break this up over multiple lines

```python
        return "%s: %s %s" % (LOG_PREFIX,
                              self.address_string(),
                              message.translate(self._control_char_table))
```

Reply:
Addressed in https://github.com/metabrainz/picard/commit/31945b31ac8e6360763a369dd05e443970a10a87 (better not rely on parent class private property).